server/src/main.rs

```diff
@@ -45,13 +51,38 @@ pub struct Context {
 
 /** Public Functions **/
 fn main() {
+    
+
+    // A layer that logs events to a file.
+    let file = File::create("debug.log");
+    let file = match file  {Ok(file) => file,Err(error) => panic!("Error: {:?}",error),};
```

Comment:
This does not seem formatted. Consider running `rustfmt` to fix issues like this.

Reply:
You could also consider just matching on the result of `File::create("debug.log")` instead of using an intermediate variable.

Reply:
Formatted, rustfmt has run

Reply:
2nd comment is also fixed.